container/raw/handler.go

```diff
@@ -112,6 +117,16 @@ func newRawContainerHandler(name string, cgroupSubsystems *libcontainer.CgroupSu
 		}
 	}
 
+	if !isRootCgroup(name) {
+		// collect network metrics only for root cgroup
+		ignoreMetrics.Add(container.NetworkUsageMetrics)
```

Comment:
Your `Copy` is really a merge, so this isn't actually creating a new copy. I'd just update the copy function to initialize a new map, and return it.